react/utils/functionsConditions.tsx

```diff
@@ -43,7 +45,9 @@ export function SimpleInputObject({
   return (
     <Input
       value={value}
-      onChange={(e: { target: { value: any } }) => onChange(e.target.value)}
+      onChange={(e: { target: { value: any } }) => {
```

Comment:
Here I think that you can change generic type to `React.ChangeEvent<HTMLInputElement>`

```suggestion
      onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
```